app/views/home/_fandoms.html.erb

```diff
@@ -0,0 +1,10 @@
+<ul>
+  <li><%= link_to t(".all_fandoms"), media_path %></li>
+  <% cache "homepage-fandoms-version1", skip_digest: true do %>
+    <% Media.for_menu.each do |medium| %>
+      <% unless medium.id.nil? %>
+        <li><%= link_to t(".#{medium.translation_key}"), medium_fandoms_path(medium) %></li>
```

Comment:
For whatever it's worth, I'm still team "don't feed this through i18n this until the caching questions from https://github.com/otwcode/otwarchive/pull/4584#discussion_r1272656196 are settled." This is no longer an active bug, since you can't set your locale on the homepage yet, but it could become a bug later if we don't opt to override the `cache` function.

(Also, I think [mobility](https://github.com/shioyama/mobility) or [globalize](https://github.com/globalize/globalize) would be a better choice for internationalizing medium names. As is, this would make it more difficult for other sites using the `otwarchive` software to use their own custom media names.)

Reply:
I don't have the spoons to look into alternate ways of translating, so I've removed the contentious commit. We can cross or burn that bridge when we come closer to it. ;)

Thank you all for reviewing!